pkg/endpoint/endpoint.go

```diff
@@ -461,6 +462,9 @@ type Endpoint struct {
 	// is enabled for this endpoint.
 	egressPolicyEnabled bool
 
+	// uuid is the UUID of the endpoint. It is assigned via GetUUID()
+	uuid string
```

Comment:
Why `uuid` for the endpoint? We already have the `id`. More left concerns in the `NewUUID`